plugins/netty/src/main/java/com/navercorp/pinpoint/plugin/netty/NettyPlugin.java

```diff
@@ -79,13 +81,28 @@ public void setup(ProfilerPluginSetupContext context) {
             transformTemplate.transform("io.netty.handler.codec.http.DefaultHttpRequest", HttpRequestTransformer.class);
             transformTemplate.transform("io.netty.handler.codec.http.HttpObjectEncoder", HttpEncoderTransformer.class);
         }
+
+        transformTemplate.transform("io.netty.util.internal.PlatformDependent", PlatformDependentTransform.class);
     }
 
     @Override
     public void setTransformTemplate(TransformTemplate transformTemplate) {
         this.transformTemplate = transformTemplate;
     }
 
+    public static class PlatformDependentTransform implements TransformCallback {
+        @Override
+        public byte[] doInTransform(Instrumentor instrumentor, ClassLoader loader, String className, Class<?> classBeingRedefined, ProtectionDomain protectionDomain, byte[] classfileBuffer) throws InstrumentException {
+            InstrumentClass target = instrumentor.getInstrumentClass(loader, className, classfileBuffer);
+
+            for (InstrumentMethod method : target.getDeclaredMethods(MethodFilters.name("usedDirectMemory", "maxDirectMemory"))) {
+                method.addInterceptor(PlatformDependentInterceptor.class);
```

Comment:
@koo-taejin I think that's not necessary.
because we don't intercept when client(I mean agent server) call `usedDirectMemory ` and `maxDirectMemory ` method.

Reply:
@yangbongsoo
I think that part is good.
If api removed, then this part can avoid unexpected situation.
(I don't think this api will be removed :)